README.md

```diff
@@ -16,11 +16,9 @@ Add the plugin to your Webpack configuration:
 
   resolve: {
     plugins: [new BowerResolvePlugin()],
-    modules: [
-        'bower_components'
-    ],
-    descriptionFiles: ['bower.json', 'package.json'],
-    mainFields: ['main', 'browser']
+    modules: ['bower_components', 'node_modules'],
+    descriptionFiles: ['.bower.json', 'package.json'],
```

Comment:
That's actually fixed in the latest version (not the most commonly used though). Still, I'd like to use `package.json` for `node_modules`, `bower.json` for `bower_components`, and webpack doesn't make it easy 😬